### PR TITLE
DashboardRow: fixes time variable not updating when used in row title

### LINKED ISSUE #33522: Row title not changes after changing a time period
**What happened**:

Row title not changes after changing a time period

image

**What you expected to happen**:

Simultaneously changing, like in other panel titles

**How to reproduce it (as minimally and precisely as possible)**:

Change row title to `from ${__from:date:DD.MM.YYYY LT} to ${__to:date:DD.MM.YYYY LT}`

image

**Anything else we need to know?**:

**Environment**:
- Grafana version: v7.5.1 (4dffb875cf)
- Data source type & version: MS SQL
- OS Grafana is installed on: Windows 10
- User OS & Browser: Chrome
- Grafana plugins:
- Others: same on v7.5.5 (b5190ee547)

### PR DESCRIPTION
**Pull request by thisisobate**
**What this PR does / why we need it**:
This PR fixes time variable not updating when used in row title.
**Which issue(s) this PR fixes**:

Fixes #33522

### CODE REVIEW

File: public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx [hunk 1]
```diff
@@ -4,29 +4,36 @@ import { Icon } from '@grafana/ui';
...
+  sub?: Unsubscribable;
   constructor(props: DashboardRowProps) {
     super(props);
 
     this.state = {
       collapsed: this.props.panel.collapsed,
     };
 
-    this.props.dashboard.on(CoreEvents.templateVariableValueUpdated, this.onVariableUpdated);
+    this.props.dashboard.on(RefreshEvent, this.onVariableUpdated);
```
Comment (hugohaggmark) [inline public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx:26]: Hmm, do we need this and the line on componentDidMount?

**Review by hugohaggmark — COMMENTED** [review]
Looks ok but I think you're creating the same subscription twice.

File: public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx (hunk 1 above)
Comment (thisisobate) [reply public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx:26]: oh...my bad.
Would fix up right away

**Comment by hugohaggmark** [comment]
Not sure if it is easy or very difficult but would be nice to add a test case for this

**Review by hugohaggmark — APPROVED** [review]
LGTM and works as expected 🎉 !

**Pull request merged**